specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/examples/DPSCreateOrUpdatePrivateEndpointConnection.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "resourceName": "myFirstProvisioningService",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0",
+    "privateEndpointConnectionName": "myPrivateEndpointConnection",
+    "privateEndpointConnection": {
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description": "Approved by johndoe@contoso.com"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/91d12660-3dec-467a-be2a-213b5544ddc0/resourceGroups/myResourceGroup/providers/Microsoft.Devices/ProvisioningServices/myFirstProvisioningService/PrivateEndpointConnections/myPrivateEndpointConnection",
+        "name": "myPrivateEndpointConnection",
+        "type": "Microsoft.Devices/ProvisioningServices/PrivateEndpointConnections",
+        "properties": {
+          "privateEndpoint": {
+            "id": "/subscriptions/a9eba280-4734-4d49-878f-b5549d1d0453/resourceGroups/networkResourceGroup/providers/Microsoft.Network/privateEndpoints/myPrivateEndpoint"
+          },
+          "privateLinkServiceConnectionState": {
+            "status": "Approved",
+            "description": "Approved by johndoe@contoso.com",
+            "actionsRequired": "None"
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/91d12660-3dec-467a-be2a-213b5544ddc0/resourceGroups/myResourceGroup/providers/Microsoft.Devices/ProvisioningServices/myFirstProvisioningService/PrivateEndpointConnections/myPrivateEndpointConnection",
+        "name": "myPrivateEndpointConnection",
+        "type": "Microsoft.Devices/ProvisioningServices/PrivateEndpointConnections",
+        "properties": {
+          "privateEndpoint": {
```

Comment:
Please add provisiongState to indicate the state of the async operation.
https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc005-provisioning-state-semantics-must-be-followed

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)